activity_log.php

```diff
@@ -30,9 +30,6 @@
 $sys = ( $is_admin && getGetValue ( 'system' ) != '' );
 
 print_header();
-
-ob_start();
```

Comment:
Great! It looks like I need to do git reset --hard <commit>. But, how do I know when your master and mine are in sync again? Especially since you've updated your master since I screwed up mine. Do I use '55d1ec6', which is the last commit you made? The manual is not real clear on that.